cdc/cdc/processor/pipeline/sorter.go

```diff
@@ -151,6 +151,30 @@ func (n *sorterNode) StartActorNode(ctx pipeline.NodeContext, eg *errgroup.Group
 						}
 					}
 
+					// We calculate memory consumption by PolymorphicEvent size.
+					size := uint64(msg.RawKV.ApproximateDataSize())
+					// NOTE we allow the quota to be exceeded if blocking means interrupting a transaction.
+					// Otherwise the pipeline would deadlock.
+					err := n.flowController.Consume(commitTs, size, func() error {
```

Comment:
Suggest add a metric to observe the blocking time.

Reply:
![image](https://user-images.githubusercontent.com/50103576/194824285-ecfcef11-dfd6-450a-8f11-5d96cd56e9b8.png)
added, thanks! PTAL~